lite/proxy/block.go

```diff
@@ -27,6 +33,9 @@ func ValidateBlock(meta *types.Block, check lite.Commit) error {
 }
 
 func ValidateHeader(head *types.Header, check lite.Commit) error {
```

Comment:
`check.Header` can also be nil and can lead to the same nil derefence, see: https://circleci.com/gh/tendermint/tendermint/4228

Reply:
@xla in deed. This PR is awaiting on
https://github.com/tendermint/tendermint/pull/1301 to be merged first
and I mention this in the commit and first Github message :) :
>Requires the following bugs to be fixed first;
>
>#1298
>#1299

Reply:
My bad! Disregard.

Reply:
And #1301 has been merged, so we should no longer have those failures in CircleCI.